### ISSUE TITLE
Error: Connection already released

### ISSUE THREAD

**Issue by werdnanoslen** [werdnanoslen/mobile-map-io#12]
```
/var/www/demos/mobile-map-io/node_modules/mysql/lib/protocol/Parser.js:77
        throw err; // Rethrow non-MySQL errors
              ^
Error: Connection already released
    at Pool.releaseConnection (/var/www/demos/mobile-map-io/node_modules/mysql/lib/Pool.js:129:13)
    at PoolConnection.release (/var/www/demos/mobile-map-io/node_modules/mysql/lib/PoolConnection.js:30:15)
    at Query._callback (/var/www/demos/mobile-map-io/api/rest.js:88:24)
    at Query.Sequence.end (/var/www/demos/mobile-map-io/node_modules/mysql/lib/protocol/sequences/Sequence.js:96:24)
    at Query._handleFinalResultPacket (/var/www/demos/mobile-map-io/node_modules/mysql/lib/protocol/sequences/Query.js:144:8)
    at Query.EofPacket (/var/www/demos/mobile-map-io/node_modules/mysql/lib/protocol/sequences/Query.js:128:8)
    at Protocol._parsePacket (/var/www/demos/mobile-map-io/node_modules/mysql/lib/protocol/Protocol.js:280:23)
    at Parser.write (/var/www/demos/mobile-map-io/node_modules/mysql/lib/protocol/Parser.js:73:12)
    at Protocol.write (/var/www/demos/mobile-map-io/node_modules/mysql/lib/protocol/Protocol.js:39:16)
    at Socket.<anonymous> (/var/www/demos/mobile-map-io/node_modules/mysql/lib/Connection.js:96:28)
error: Forever detected script exited with code: 8
error: Script restart attempt #2
mobile-map-io API running at port 8080
```

**Comment by werdnanoslen** [comment]
```
/var/www/demos/mobile-map-io/node_modules/mysql/lib/protocol/Parser.js:77
        throw err; // Rethrow non-MySQL errors
              ^
Error: Connection already released
    at Pool.releaseConnection (/var/www/demos/mobile-map-io/node_modules/mysql/lib/Pool.js:129:13)
    at PoolConnection.release (/var/www/demos/mobile-map-io/node_modules/mysql/lib/PoolConnection.js:30:15)
    at Query._callback (/var/www/demos/mobile-map-io/api/rest.js:88:24)
    at Query.Sequence.end (/var/www/demos/mobile-map-io/node_modules/mysql/lib/protocol/sequences/Sequence.js:96:24)
    at Query._handleFinalResultPacket (/var/www/demos/mobile-map-io/node_modules/mysql/lib/protocol/sequences/Query.js:144:8)
    at Query.EofPacket (/var/www/demos/mobile-map-io/node_modules/mysql/lib/protocol/sequences/Query.js:128:8)
    at Protocol._parsePacket (/var/www/demos/mobile-map-io/node_modules/mysql/lib/protocol/Protocol.js:280:23)
    at Parser.write (/var/www/demos/mobile-map-io/node_modules/mysql/lib/protocol/Parser.js:73:12)
    at Protocol.write (/var/www/demos/mobile-map-io/node_modules/mysql/lib/protocol/Protocol.js:39:16)
    at Socket.<anonymous> (/var/www/demos/mobile-map-io/node_modules/mysql/lib/Connection.js:96:28)
error: Forever detected script exited with code: 8
error: Script restart attempt #1
mobile-map-io API running at port 8080
```

**Comment by werdnanoslen** [comment]
solved with revert 8ea6d79856d974bb571f7a49105894a154abcf22
